quickstarts/Embeddings.ipynb

```diff
@@ -35,18 +35,34 @@
     },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Note that the output_dimensionality parameter for embeedings is not in 0.5.1 but will be in the next release.
</pre><p> </p><p>If you are going to merge this before it's released, I would do something like:</p><p> </p><p><code>pip install 'google-generativeai&gt;0.5.1'</code></p><p> </p><p>So that users get a released version (safer) and it doesn't require a post-update.</p><p> </p><p>But this is OK too if you plan on updating after release (maybe use the new URL though?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/82/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>File uploads broke yesterday, so I pushed a 0.5.2:</p><p> </p><p><a href="https://pypi.org/project/google-generativeai#history" target="_blank">https://pypi.org/project/google-generativeai/#history</a></p><p> </p><p><a href="https://github.com/google-gemini/generative-ai-python/issues/292" target="_blank">https://github.com/google-gemini/generative-ai-python/issues/292</a></p>

Reply:
<p>So a standard install is fine now.</p>